.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,19 @@
+## Source: https://github.com/marketplace/actions/ansible-lint
```

Comment:
If you rename your workflow `tests.yml` the badge to README.md can be added with:

```md
|Tests|

.. |Tests| image:: https://github.com/aalaesar/install_nextcloud/workflows/Tests/badge.svg
   :target: https://github.com/aalaesar/install_nextcloud/actions?workflow=Tests
   :alt: Tests
```

Reply:
I've added ansible-lint and yamllint as two separate workflows so I've added two badges to README.